client/src/actions/request.js

```diff
@@ -0,0 +1,17 @@
+import axios from "axios";
+// import { REQUEST_SENT } from "./types";
```

Comment:
Why is this commented out? Is it future integration or can it be removed

Reply:
I've removed it. It was there in case I needed to update the state when a request was sent but it is not needed.